docs/sql-guide/statements/statement-insert-bulk.md

```diff
@@ -28,6 +28,7 @@ Supported formats include:
 * CSV
 * PARQUET
 * NDJSON
+* ORC
```

Comment:
I'm going to tweak this list and get it into a Before you begin heading because the ORC data type isn't explained or ferenced anywhere else but here. NDJSON and Parquet could also do with background reading too :)